core/capabilities/transmission/local_target_capability.go

```diff
@@ -36,7 +38,7 @@ func (l *LocalTargetCapability) Execute(ctx context.Context, req capabilities.Ca
 
 	peerIDToTransmissionDelay, err := GetPeerIDToTransmissionDelay(l.don.Members, req)
 	if err != nil {
-		return nil, fmt.Errorf("failed to get peer ID to transmission delay map: %w", err)
+		return nil, fmt.Errorf("capability id: %s failed to get peer ID to transmission delay map: %w", l.capabilityID, err)
```

Comment:
Nit: I would  expect the caller to log/wrap the error with something like the following rather than having that information in the returned error

fmt.Errorf("failed to create local target capability %s: %w", capabilityID, "err")

Reply:
I resolved this by creating custom workflow engine errors. This way any higher level code that's handling errors can extract the required fields and have them as key / value pairs in labels